spec/proposed/character_factory_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../../lib/proposed/character'
+require_relative '../../lib/proposed/character_factory'
+require_relative '../../lib/proposed/character_validator'
+
+describe CharacterFactory do
```

Comment:
Testujemy tylko i wyłącznie fabrykę i to czy odpowiednio buduje character i czy woła validator, nic więcej nas w fabryce nie obchodzi, bo odpowiedzialność za walidację jest przerzucona na klasę validatora.